### PR TITLE
Rename Math::stepify to snapped

### PR DESCRIPTION
**Pull request by madmiraal**
As originally identified [here](https://github.com/godotengine/godot/issues/16863#issuecomment-655258916), and as described in the @GDScript [documentation](https://docs.godotengine.org/en/stable/classes/class_@gdscript.html#class-gdscript-method-stepify), `Math::stepify()` returns a snapped value. Furthermore, this is what it is called in the [`Vector2`](https://docs.godotengine.org/en/stable/classes/class_vector2.html#class-vector2-method-snapped) and [`Vector3`](https://docs.godotengine.org/en/stable/classes/class_vector3.html#class-vector3-method-snapped) versions that use this method. Therefore, this PR renames `Math::stepify()` to `stepped()`.

Part of #16863.

### CODE REVIEW

**Comment by madmiraal** [comment]
Rebased following merge of #34193 and #44699.

File: modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs [hunk 1]
```diff
@@ -618,10 +618,10 @@ public static int StepDecimals(real_t step)
         /// This can also be used to round a floating point
         /// number to an arbitrary number of decimals.
         /// </summary>
-        /// <param name="s">The value to stepify.</param>
+        /// <param name="s">The value to snap.</param>
         /// <param name="step">The step size to snap to.</param>
         /// <returns></returns>
-        public static real_t Stepify(real_t s, real_t step)
+        public static real_t Snapped(real_t s, real_t step)
```
Comment (aaronfranke) [inline modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs:624]: The change in this file is good, except that you forgot to change Vector2.cs and Vector3.cs which use this method.

**Review by aaronfranke — CHANGES_REQUESTED** [review]

File: modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs (hunk 1 above)
Comment (madmiraal) [reply modules/mono/glue/GodotSharp/GodotSharp/Core/Mathf.cs:624]: Done.

**Review by akien-mga — APPROVED** [review]

**Comment by aaronfranke** [comment]
I haven't compiled/tested this but it looks good. All instances of `Stepify` in C# are now `Snapped`.

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**